pkg/envoy/route/route_config_test.go

```diff
@@ -267,43 +265,157 @@ func TestBuildRoute(t *testing.T) {
 		pathRegex        string
 		method           string
 		headersMap       map[string]string
+		expectedRoute    *xds_route.Route
 	}{
 		{
-			name:             "default",
-			pathRegex:        "/somepath",
-			method:           "GET",
-			headersMap:       map[string]string{"hello": "goodbye", "header1": "another-header"},
-			totalWeight:      100,
-			direction:        OutboundRoute,
-			weightedClusters: weightedClusters,
+			name:        "outbound route",
+			pathRegex:   "/somepath",
+			method:      "GET",
+			headersMap:  map[string]string{"header1": "header1-val", "header2": "header2-val"},
+			totalWeight: 100,
+			direction:   OutboundRoute,
+			weightedClusters: set.NewSetFromSlice([]interface{}{
+				service.WeightedCluster{ClusterName: service.ClusterName("osm/bookstore-1"), Weight: 30},
```

Comment:
Not really what I wanted, because each test case in the table of tests can have different inputs. Its pretty normal to hardcode inputs to tests while using table driven tests. Otherwise one would need to define variables for every different input that is different in each test case, which is far from ideal.